jobbergate-agent/jobbergate_agent/jobbergate/submit.py

```diff
@@ -244,7 +244,7 @@ async def _reject_handler(params: DoExceptParams) -> None:
         logger.debug(f"Using local slurm user '{username}' for job submission")
         subprocess_handler = SubprocessAsUserHandler(username)
 
-    submit_dir = pending_job_submission.execution_directory or SETTINGS.DEFAULT_SLURM_WORK_DIR
+    submit_dir = pending_job_submission.execution_directory or Path(f"/home/{username}")
```

Comment:
This assumes that the base home directory is at `/home`, which I think it's not a fair assumption. I'd say we should probably create a setting with this value defaulting to `/home`, so it is customizable if needed.

Reply:
Following what Matheus suggested, we could keep `SETTINGS.DEFAULT_SLURM_WORK_DIR`, set its default to `"/home/{username}"`, and format it at submission time `SETTINGS.DEFAULT_SLURM_WORK_DIR.format(username=username)`.

Reply:
No, we should remove the `DEFAULT_SLURM_WORK_DIR`, I think. We can use `Path.home()` instead. This should always return a valid directory according to the Python docs and POSIX specs:

https://docs.python.org/3/library/pathlib.html#pathlib.Path.home
https://docs.python.org/3/library/os.path.html#os.path.expanduser
https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html (see entry for Other Environment Variables > HOME)